navbar.js

```diff
@@ -31,6 +31,11 @@ module.exports = {
       label: 'Ideas',
       to: 'https://ideas.sailpoint.com',
     },
+    {
+      position: 'left',
+      label: 'Ambassador Program',
+      to: 'https://developer.sailpoint.com/ambassadors/',
```

Comment:
You might want to change it to just `/ambassadors` that way links for PRs and localhost still work

Reply:
fixed